test/documentViewer/pdfViewer/pdfjsViewer/test.js

```diff
@@ -20,13 +20,13 @@
  */
 define([
     'jquery',
-    'core/promise',
+    
```

Comment:
Could you also please remove the blank line here and in the couple of next files